File: app/src/main/java/com/example/sample/currencyapp/data/ApiRequests.java
package com.example.sample.currencyapp.data;

import com.example.sample.currencyapp.data.model.Currencies;
import io.reactivex.Single;
import retrofit2.http.GET;

/**
 * Created by Mostafa on 08/20/2017.
 */

public interface ApiRequests {

    @GET("currencies.json")
    Single<Currencies> getCurrencies();
}

File: app/src/main/java/com/example/sample/currencyapp/data/ApiRequests.kt
package com.example.sample.currencyapp.data

import com.example.sample.currencyapp.data.model.RatesResponse
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query


interface ApiRequests {
    @GET("latest")
    fun changeRate(@Query("base") base: String): Single<RatesResponse>
}
